.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -44,29 +41,36 @@ body:
   attributes:
     label: Issue description
     description: |
-      Describe your issue briefly. What doesn't work, and how do you expect it to work instead?
+      Describe your issue briefly.
+
       You can include images or videos with drag and drop, and format code blocks or logs with <code>```</code> tags, on separate lines before and after the text. (Use <code>```gdscript</code> to add GDScript syntax highlighting.)
-      Please do not add code examples or error messages as screenshots, but as text, this helps searching for issues and testing the code. If you are reporting a bug in the editor interface, like the script editor, please provide both a screenshot *and* the text of the code to help with testing.
+
```

Comment:
I just had an epiphany. I noticed that providing videos/GIFs is so much easier than creating a MRP. Perhaps this should be encouraged more? Not just "You can", rather "You should".

Reply:
A video is nice to have, but an MRP is always better. The MRP can be used by a contributor who works on a fix to both confirm the bug, and confirm the fix. If you just have a video, you have to create the MRP yourself by following the video, which can be quite tedious.

Reply:
Of course, not completely ruling out a MRP at all. But a video is **extremely** helpful when possible, I feel.

Reply:
A short video is nice, but we often have bug reports where it's just a 2-min video with no explanations, and those are really hard to work with. A MRP with a list of what to do with it to trigger the bug is the best scenario.